### DISCUSSION TITLE
Error when trying to run GBG example dataset

### DISCUSSION THREAD

**Discussion by JJStobart** [UMEP-dev/UMEP#460] [Python]
Hello! I am trying to run the example dataset for GBG. I am using the latest QGIS 3.26.3 and have installed as per the "getting started" page. I can create SVF and Wall aspect and height grids. When i try to run the SOLWEIG model i get the following error: 

2022-09-21T11:33:08     CRITICAL    EXCEPTION IN C:\Users/SEVEWO/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\UMEP\SOLWEIG\solweigworker.py, 
             LINE 290 "dRad, Kside = so.Solweig_2022a_calc(" 
             ERROR MESSAGE: local variable 'patches_in_band' referenced before assignment

Any help would be greatly appriciated! I am using default values just to test the model, following a tutorial on youtube. 

Thanks
Joe

**Comment by biglimp** [comment]
We just updated to a new version of UMEP 3.20.6. Should be availabe now or very soon within QGIS plugins

**Reply by JJStobart** [reply]
wow! thank you for the very quick reply - have updated and tried again and it works! thank you very much again. 
Cheers,
Joe
